controller/update_pools.go

```diff
@@ -0,0 +1,52 @@
+package controller
+
+import (
+	"github.com/blockassets/bam_agent/service"
+	"net/http"
+)
+
+// command to update the ip address for
+// pools
+// POST
+//Just POST this file:
+//{
+//"pool1":"",
+//"pool2":"",
+//"pool3":""
+//}
+// eg { "pool1":"111.2.3.4", "pool2":"112.3.4.5", "pool3":"113.4.5.6"}
+// and we update the conf.default file on the miner
+
+const defConfigPath = "usr/app/conf.default"
+
+// Implements Controller interface
+type UpdatePoolsCtrl struct {
+}
+
+func (c UpdatePoolsCtrl) build() *Controller {
+	return &Controller{
+		Methods: []string{http.MethodPost},
```

Comment:
PUT please.

https://stackoverflow.com/questions/630453/put-vs-post-in-rest

Reply:
:-)